### DISCUSSION TITLE
deep learning, machine learning, software dev, hardware dev

### DISCUSSION THREAD

**Discussion by babooloo** [WeAreMahsaAmini/Core#449] [Introductions]
My main expertise is deep learning, but I have 12 years of experience working on everything nerdy from embedded devices (both hardware and software) to web development (frondend and backend). 
Ready to get involved ...

**Comment by tcf-admin** [comment]
Hi @babooloo, 

in forDem, an e-democracy platform under development, we heavily rely on AI and algorithms to categorize the content, suggest similar initiatives, expressions etc. to our users.
We also categorize, cluster, and import user content on different social media and while doing so we need to sort it. 
Do you think we can benefit from your expertise? if so, drop a note here: https://github.com/tcfev/forDem/discussions/52
